test/http2.js

```diff
@@ -67,6 +67,65 @@ test('Should support H2 connection', async t => {
   t.equal(Buffer.concat(body).toString('utf8'), 'hello h2!')
 })
 
+test('Should support H2 connection(multiple requests)', async t => {
+  const server = createSecureServer(pem)
+
+  server.on('stream', async (stream, headers, _flags, rawHeaders) => {
+    t.equal(headers['x-my-header'], 'foo')
+    t.equal(headers[':method'], 'POST')
+    const reqData = []
+    stream.on('data', chunk => reqData.push(chunk.toString()))
+    await once(stream, 'end')
+    const reqBody = reqData.join('')
+    t.equal(reqBody.length > 0, true)
+    stream.respond({
+      'content-type': 'text/plain; charset=utf-8',
+      'x-custom-h2': 'hello',
+      ':status': 200
+    })
+    stream.end(`hello h2! ${reqBody}`)
+  })
+
+  server.listen(0)
+  await once(server, 'listening')
+
+  const client = new Client(`https://localhost:${server.address().port}`, {
+    connect: {
+      rejectUnauthorized: false
+    },
+    allowH2: true
+  })
+
+  t.plan(21)
+  t.teardown(server.close.bind(server))
+  t.teardown(client.close.bind(client))
+
+  for (let i = 0; i < 3; i++) {
+    const sendBody = `seq ${i}`
+    const body = []
+    const response = await client.request({
+      path: '/',
+      method: 'POST',
+      headers: {
+        'content-type': 'text/plain; charset=utf-8',
+        'x-my-header': 'foo'
+      },
+      // FIXME: If send a Buffer or string, the stream will not end, causing it to get stuck.
```

Comment:
This is already done in #2336 right?

Reply:
yes

Reply:
Can we remove it if solved already?

Reply:
Will I remove it after #2336 merges, or should I remove it now?

Reply:
please remove